grafana_wtf/core.py

```diff
@@ -492,6 +492,40 @@ def explore_dashboards(self, with_data_details: bool = False):
 
         return results
 
+    def plugins_list(self):
+        return self.grafana.plugin.get_installed_plugins()
+
+    def plugins_status(self):
+        status = []
+        plugins = self.grafana.plugin.get_installed_plugins()
+        for plugin in plugins:
+            plugin = munchify(plugin)
+            item = Munch(
```

Comment:
Wow I haven't used `munchify` ever , that seems like a great way to remove the tedious key value mapping with strings.

Reply:
If you can afford the runtime speed bump, it definitively saves a few keystrokes!

By the way, big thanks to @dsc, @vmalloc, @JosePVB, @innermatrix, @d1618033, and all others who conceived it, and are maintaining it.